x-pack/plugins/ml/server/models/data_frame_analytics/validation.ts

```diff
@@ -143,7 +156,7 @@ function getRegressionAndClassificationMessage(
           id: 'num_top_classes',
           text: i18n.translate('xpack.ml.models.dfaValidation.messages.topClassesSuccessMessage', {
             defaultMessage:
-              'Probabilities will be reported for {numCategories, plural, one {# category} other {# categories}}.',
+              'The dependent variable has {numCategories, plural, one {# unique class} other {# unique classes}}. Probabilities will be reported for each of these values.',
```

Comment:
I'm not convinced this message needs changing at all, as this is misleading - the dependent variable doesn't have 1 unique class, it's just I've chosen to only report probabilities for 1 category. Or maybe just add `Predicted` to the start of the message, so it is consistent with the 'All classes' case above:

![image](https://user-images.githubusercontent.com/7405507/115380195-40596c00-a1ca-11eb-8bf9-b366f3b1dc0e.png)